_test/test_setup.go

```diff
@@ -29,7 +29,7 @@ import (
 func spinUpTestChains(t *testing.T, testChains ...testChain) relayer.Chains {
 	var (
 		resources []*dockertest.Resource
-		chains    = make([]*relayer.Chain, len(testChains))
+		chains    = make(relayer.Chains)
```

Comment:
You can still use `make(type, len(x))` for types that are defined as slices or maps, i.e. this is okay to be `make(relayer.Chains, len(testChains))`.

Reply:
@mark-rushakoff can't seem to get it to pass the chain integration tests after making this change.

Reply:
No worries, it can stay as is.